src/SUMMARY.md

```diff
@@ -6,6 +6,7 @@
 
 - [Part 1: Building, debugging, and contributing to Rustc](./part-1-intro.md)
     - [About the compiler team](./compiler-team.md)
+    - [Getting Started](./getting-started.md)
```

Comment:
Move this to be the first item? Since it's about getting started, that would make the most sense to my mind.

Reply:
Yeah, I agree, but we're probably going to reorganize the whole first part soon.

Reply:
I just went ahead and did this. We can rearrange later.